### PR TITLE
sys-apps/rng-tools: add 6.13

### PR DESCRIPTION
**Pull request by thesamesam**
Signed-off-by: Sam James <sam@gentoo.org>

### CODE REVIEW

File: sys-apps/rng-tools/rng-tools-6.13.ebuild [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+src_prepare() {
+	echo 'bin_PROGRAMS = randstat' >> contrib/Makefile.am || die
+
+	default
+
+	mv README.md README || die
+
+	eautoreconf
+
+	sed -i '/^AR /d' Makefile.in || die
```
Comment (SoapGentoo) [inline sys-apps/rng-tools/rng-tools-6.13.ebuild:47]: any chance of fixing this properly upstream?

File: sys-apps/rng-tools/rng-tools-6.13.ebuild [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+		local DISABLE_AUTOFORMATTING=1
+		local DOC_CONTENTS="
+The PKCS11 entropy source may require extra packages (e.g. 'dev-libs/opensc')
+to support various smartcard readers. Make sure 'PKCS11_OPTIONS' in:
+	'${EPREFIX}/etc/conf.d/rngd'
+reflects the correct PKCS11 engine path to be used by rngd.
+"
+		readme.gentoo_create_doc
+	fi
+
```
Comment (SoapGentoo) [inline sys-apps/rng-tools/rng-tools-6.13.ebuild:78]: superfluous empty line

File: sys-apps/rng-tools/rng-tools-6.13.ebuild [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	)
+
+	econf "${myeconfargs[@]}"
+}
+
+src_install() {
+	default
+	newinitd "${FILESDIR}"/rngd-initd-6.11 rngd
+	newconfd "${FILESDIR}"/rngd-confd-6.11 rngd
+	systemd_dounit "${S}"/rngd.service
```
Comment (SoapGentoo) [inline sys-apps/rng-tools/rng-tools-6.13.ebuild:66]:
```suggestion
	systemd_dounit rngd.service
```

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sys-apps/rng-tools/rng-tools-6.13.ebuild (hunk 1 above)
Comment (thesamesam) [reply sys-apps/rng-tools/rng-tools-6.13.ebuild:47]: It's already fixed!

**Pull request closed without merging**